perf/Manifest.toml

```diff
@@ -101,9 +101,9 @@ version = "0.4.2"
 
 [[deps.BandedMatrices]]
 deps = ["ArrayLayouts", "FillArrays", "LinearAlgebra", "PrecompileTools", "SparseArrays"]
-git-tree-sha1 = "6e9b384d5e21452bc81cae3552c74f430e320985"
+git-tree-sha1 = "106122bad983acd8e21aef3f7a7f91a3ac739077"
 uuid = "aae01518-5342-5314-be14-df237901396f"
-version = "0.17.35"
+version = "0.17.36"
```

Comment:
Are all these other package updates automatically triggered when you `] up ClimaLSM`? Or you updated the entire environment?

Reply:
Ahh good point, I'll revert them and just up ClimaLSM